src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.spec.ts

```diff
@@ -79,37 +79,37 @@ describe('InstructorCourseEditPageComponent', () => {
   });
 
   it('should not change course details if CANCEL is requested', () => {
-    spyOn(courseService, 'getCourseAsInstructor').and.returnValue(of({
+    component.course = {
       courseId: 'exampleId',
       courseName: 'Example Course',
       timeZone: 'UTC (UTC)',
       creationTimestamp: 0,
       deletionTimestamp: 1000,
```

Comment:
For test data like this, it is better to define them as `const` to avoid repeatition in test suites.

You can refer to other merged front-end testing (e.g. #10935) on how to refactor this.